src/main/resources/application.yml

```diff
@@ -15,8 +15,8 @@ org:
       logging:
         LoggingSystem: none
 jar:
-  path:
```

Comment:
I guess this is not required now as there is no code handling done by us, Will remove this change